.github/workflows/deploy.yml

```diff
@@ -27,17 +27,33 @@ jobs:
   deploy:
     name: Deploy
     runs-on: ubuntu-latest
+    container:
+        image: ubuntu:latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: ruby/setup-ruby@v1
+      - name: Update packages
```

Comment:
```suggestion
      - name: Update package list
```

Reply:
I agree with this change - it is a more accurate description of what `apt-get update` does.